### PR TITLE
domlopluis94

### PR DESCRIPTION
**Pull request by domlopluis94**

### CODE REVIEW

**Comment by LuciaMm** [comment]
Good job!

File: Assignment2/domlopluis94-LuisDominguez.rdf [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?xml version='1.0' encoding='utf-8' ?>
+<rdf:RDF xmlns:ns0='http://' xmlns:rdf='http://www.w3.org/1999/02/22-rdf-syntax-ns#' xmlns:xsd='http://www.w3.org/2001/XMLSchema#'>
+<rdf:Description rdf:about='http://class01'>
```
Comment (ocorcho) [inline Assignment2/domlopluis94-LuisDominguez.rdf]: The URIs are not correctly constructed

File: Assignment2/domlopluis94-LuisDominguez.ttl [hunk 2]
```diff
@@ -0,0 +1,18 @@
+@prefix ex: <http://> .
```
Comment (ocorcho) [inline Assignment2/domlopluis94-LuisDominguez.ttl]: You should improve this namesapce.

File: Assignment2/domlopluis94-LuisDominguez.ttl [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+ex:Measurament8401  
+ ex:hasTemperature "29"^^xsd:number ;
+ ex:atTime "2010-06-12T12:00:12"^^xsd:date.
+
+ex:Computer101
+  ex:hasOwner ex:User10A.
+
+ex:User10A
+ ex:hasName "Pedro"^^xsd:name.
```
Comment (ocorcho) [inline Assignment2/domlopluis94-LuisDominguez.ttl]: the datatype name does not exist. It is xsd:string

**Comment by ocorcho** [comment]
Check my comments

**Pull request closed without merging**